CONTRIBUTING.md

```diff
@@ -1,139 +1,138 @@
+8. Link your pull request to the issue you created. This important step helps us know which issue this solution fixes. Tip: You can link the pull request in the body of the pull request template using the GitHub comment `closes #issue-no` or `resolves #issue-no`. 
+9. Submit your pull request against the `develop` branch. 
 
-## Our use of branches
+If the pull request is accepted, we will schedule the issue and merge the pull request for you. 
 
-See the [release documentation](https://github.com/uswds/uswds/wiki/Release-process) for more information on our git/GitHub release workflow.
+#### If it's a feature request or enhancement:
+
+1. Check our [open issues backlog](https://github.com/uswds/uswds/issues) for any duplicate or similar issues.
+2. If your idea has already been suggested and fulfills your business need, upvote that feature request with a thumbs up emoji (👍) and comment on the issue to let us know why you need this feature request or enhancement and any other supporting information. 
+Tip: If you want to find other feature requests open for upvoting, check out our [feature request view sorted by status](https://github.com/orgs/uswds/projects/8/views/18?sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Status). 
+3. If your proposed fix is not in the [open issues backlog](https://github.com/uswds/uswds/issues), [create an issue](https://github.com/uswds/uswds/issues/new?assignees=&labels=type%3A+feature+request&template=feature_request.md&title=) describing your proposal. This doesn’t mean we don’t want you to create a pull request. We simply want to start the process with an online conversation first. Plus, other community members might have supporting thoughts to add to your proposal. If you’ve already got a pull request, no worries. Go ahead and attach it to the issue. 
+
+
+### ✨ Proposing something else?
+
+If you’d like to contribute something else that doesn’t fall into any of the above, we’d still love to hear about it. Just create an issue or a discussion, and we can talk about it.  
+
+## How we prioritize
+
+Once you’ve submitted a contribution, we triage them based on the following considerations:
+1. Size: Can we accomplish this in a sprint or will this take longer? 
+2. Severity: What type of functionality is impacted? Is there a workaround?
+3. Priority: Does this align with our vision and roadmap goals? 
+
+Note: We prioritize issues impacting accessibility. 
+
+These considerations help us decide if and when we can work on the issue. If the issue is accepted, we scheduled them for an upcoming sprint (a 2-week work period).
+
+You can stay up to date on the status of your contributions through [GitHub email notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) (external link) and the assigned labels on the issue. 
```

Comment:
Do we need to identify external links on github? 

```suggestion
You can stay up to date on the status of your contributions through [GitHub email notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) and the assigned labels on the issue. 
```

Reply:
Yes, we do! Since its not pointing to our _own_ GitHub.